tests/components/arve/conftest.py

```diff
@@ -0,0 +1,45 @@
+"""Common fixtures for the Arve tests."""
+
+from collections.abc import Generator
+from unittest.mock import AsyncMock, MagicMock, patch
+
+import pytest
+
+from homeassistant.components.arve.const import DOMAIN
+from homeassistant.core import HomeAssistant
+
+from . import USER_INPUT
+
+from tests.common import MockConfigEntry
+
+
+@pytest.fixture
+def mock_setup_entry() -> Generator[AsyncMock, None, None]:
+    """Override async_setup_entry."""
+    with patch(
+        "homeassistant.components.arve.async_setup_entry", return_value=True
+    ) as mock_setup_entry:
+        yield mock_setup_entry
+
+
+@pytest.fixture
+def mock_config_entry(hass: HomeAssistant, mock_arve: MagicMock) -> MockConfigEntry:
+    """Return the default mocked config entry."""
+    entry = MockConfigEntry(title="Arve", domain=DOMAIN, data=USER_INPUT, unique_id=1)
+    entry.add_to_hass(hass)
+    return entry
```

Comment:
I'd say just return only the mock config entry and have the function in `__init__.py` add the entry to hass